ui/src/Products/Product.ts

```diff
@@ -52,4 +53,33 @@ const unassignedProductName = 'unassigned';
 
 export function isUnassignedProduct(product: Product): boolean {
     return product.name === unassignedProductName;
-}
\ No newline at end of file
+}
+
+export function isActiveProduct(product: Product, viewingDate: Date): boolean {
+    return product.name.toLowerCase() !== 'unassigned'
+        && !product.archived
+        && (product.endDate == null || product.endDate >= moment(viewingDate).format('YYYY-MM-DD'));
+}
+
+export  function isProductMatchingSelectedFilters(product: Product, locationTagFilters: Array<string>, productTagFilters: Array<string>): boolean {
+    let isMatchingLocationFilter = false;
+    let isMatchingProductTagFilter = false;
+
+    if ((product.spaceLocation && locationTagFilters.includes(product.spaceLocation.name))
+        || locationTagFilters.length === 0) {
+        isMatchingLocationFilter = true;
```

Comment:
If (expression) is true set variable to true, else set variable to false ->
set variable to (expression)